smriprep/interfaces/surf.py

```diff
@@ -120,14 +145,42 @@ def normalize_surfs(in_file, transform_file, newpath=None):
         pointset.meta.setdefault("AnatomicalStructureSecondary", "MidThickness")
         pointset.meta.setdefault("GeometricType", "Anatomical")
 
+    # FreeSurfer incorrectly uses "Sphere" for spherical surfaces
+    if pointset.meta.get("GeometricType") == "Sphere":
+        pointset.meta["GeometricType"] = "Spherical"
```

Comment:
This got left in when I decided to make a new interface instead of reusing this one. I could remove it, but it's also not wrong...